conclave-host/src/main/kotlin/com/r3/conclave/host/EnclaveHost.kt

```diff
@@ -414,8 +415,9 @@ class EnclaveHost private constructor(
         hostStateManager.checkStateIsNot<Closed> { "The host has been closed." }
 
         // This can throw IllegalArgumentException which we don't want wrapped in a EnclaveLoadException.
-        attestationService = AttestationServiceFactory.getService(enclaveMode, attestationParameters)
-        quotingService = EnclaveQuoteServiceFactory.getService(attestationParameters?.takeIf { enclaveMode.isHardware })
+        attestationService = AttestationServiceFactory.getService(enclaveMode, attestationParameters, enclaveHandle)
+        quotingService =
```

Comment:
Add a TODO that this will need to fixed properly when Gramine attestation is introduced.